client/assets/components/navbar.scss

```diff
@@ -2,10 +2,10 @@
 @import '../colors.scss';
```

Comment:
The focus of this PR is to change log file location and not editing or updating the client side of the application. Please revert.